### ISSUE TITLE
Center-Aligned Combo Chart: Hover-over doesn't display correctly when resized

### ISSUE THREAD

**Issue by AlexMcAndrew** [google/google-visualization-issues#2513]
This is only an issue when using a centered aligned div for the graph. Issue can be demonstrated with the code here:

[https://jsfiddle.net/es4a638r/3/](https://jsfiddle.net/es4a638r/3/)

Once loaded, hover over the bars and line points and the hover over display will appear correctly. However, if you resize the center line, which separates the code section from the display section, like this:

![moveseperator](https://user-images.githubusercontent.com/29785290/27711015-c9ca7fa8-5d75-11e7-88f0-0a14d238fe92.png)

...and then hover over the graph points you'll notice that;

1. Hovering over on the bars still works fine.
2. Hovering over the line no longer appears, when directly over the top of the point.

Even stranger, the hover points for the line do display, but the 'trigger point' has moved horizontally (refer to below screenshot), meaning that it won't display if you hover directly over the top of the point.

![combochartissue](https://user-images.githubusercontent.com/29785290/27710951-8c7c5d7e-5d75-11e7-8fa0-d6c542661247.png)

As I mentioned above, this seems to only be an issue with a center-aligned div:
`<div id="EscStage" align="center"></div>`

The issue doesn't present itself when alignment is set to anything else.

**Comment by dlaliberte** [comment]
Yes, there are many similar problems if you add margin, border, padding, or other positioning or sizing styles to the container element.  If you want to add such things, it usually works to wrap your container element with another element where you apply those styles.

**Comment by ShaneBowen1** [comment]
Did you get any solution to this problem. Mainly the part about triggering the tooltip. Thanks
